TESTING.md

```diff
@@ -0,0 +1,25 @@
+## Testing code that uses Google Cloud Storage
+
+`RemoteStorageHelper` contains convenience methods to make setting up and cleaning up the test buckets easier. To use this class:
```

Comment:
This isn't a complete story for Storage but it does resolve the broken link. I think there's an additional AI here to flesh this file out more?